instaclone/src/App.js

```diff
@@ -2,35 +2,44 @@ import React, { Component } from 'react';
 import SearchBar from './components/SearchBar';
 import PostContainer from "./components/PostContainer"
 import dummydata from "./components/dumyData";
-
 import "./App.css"
 class App extends Component {
   constructor (props){
     super(props);
     this.state ={
-        data : dummydata,
-        message : ""
+        data : [],
+        message : "",
     }
     this.title = "Instagram"
   }
 
+  componentDidMount (){
+    this.setState({
+      data : dummydata
+    })
+  }
+
 
+  handleIncrementCount = ()=>{
```

Comment:
It doesn't appear that this method is doing anything. Should clear it to clean up code.